include/eve/bundle.hpp

```diff
@@ -0,0 +1,77 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/kumi.hpp>
+#include <eve/traits/cardinal.hpp>
+#include <eve/traits/element_type.hpp>
+#include <eve/concept/vectorized.hpp>
+
+namespace eve
+{
+  template<simd_value... Members> struct bundle;
+
+  template<simd_value Member, simd_value... Members>
+  requires( (Member::size() == Members::size()) && ... && true )
```

Comment:
Wait.

What `size` this is. Is it `wide::size()` or `bundle::size()` because `bundle::size()` is `tuple_size`.

Probably `cardinal`?

Reply:
cardinal_v is the correct thing